examples/api_gateway/scaler.c

```diff
@@ -212,19 +204,11 @@ move_buffer_to_container(void) {
                  * Add rx pipe to epoll for polling
                  */
 
-                // TODO: @bdevierno1 need to modify this dequeue function to push buffered packets to tx_pipe
-                // push all buffered packets to the pipe
-                if (dequeue_and_free_buffer_map(flow, ring, pipe->tx_pipe) < 0) {
+                int rx_fd = 0; // TODO: get the corresponding rx_fd number.
+                if (dequeue_and_free_buffer_map(flow, pipe->tx_pipe, rx_fd) < 0) {
```

Comment:
Probably want it to be this.
```suggestion
                if (dequeue_and_free_buffer_map(flow, pipe->tx_pipe, pipe->rx_pipe) < 0) {
```